Makefile

```diff
@@ -93,4 +93,4 @@ docker-down:
 	docker-compose -f ./app/docker-compose.yml down
 
 cypress-open:
-	npm --prefix ./app run cypress
+	TZ=GMT npm --prefix ./app run cypress
```

Comment:
Will this work with the BST time change ?

Reply:
Annoyingly, I don't believe there is a way to enforce the timezone within Cypress config other than setting the 'TZ' environment variable.

I think we would have to add this as an environment variable into the github action pipeline jobs in order to ensure that the timezone is always aligned between local and pipeline runs.

Failing that, I could just amend the test assertion slightly so that it doesn't assert on the 'lastUpdated' time.

Any thoughts/suggestions?

Reply:
Yeah ok..yes we need to look into the options of the env variable in github actions or even part of scripts section in package.json. we can live this for now.